### PR TITLE
[ME-37] Fixes iOS Share Sheet invocation

### PR DESCRIPTION
**Pull request by ashfurrow**
Couldn't help myself, my curiosity got the better of me.

This bug runs deeper than Airdrop, but that's where we saw it first. Users sharing artworks to other iOS or macOS devices were sending a _text file_, which is *not* QWoA at all. So what was going on?

Our previous work on sharing had us doing a lot of semantic construction before passing the content on to iOS to actually share. This is an anti-pattern; the iOS share API is designed to be given a URL, title, and message so that the OS can massage that data into whatever semantic representation makes sense for the share destination the user picks.

That is to say, instead of computing a string `title` with the artwork title, artist names, and URL all in that string, we should be sending an object with that data to let the OS render however it makes sense.

I've verified this works on-device.

### CODE REVIEW

File: src/lib/Scenes/Artwork/Components/ArtworkActions.tsx [hunk 1]
```diff
@@ -26,22 +27,19 @@ interface ArtworkActionsProps {
...
+  let computedTitle: string | null
   if (artists && artists.length) {
-    const names = []
-    artists.forEach((artist, i) => {
-      if (i < 3) {
-        names.push(artist.name)
-      }
-    })
-    message = `${title} by ${names.join(", ")} on Artsy https://artsy.net${href}`
+    const names = take(artists, 3).map(arist => arist.name)
```
Comment (kierangillen) [inline src/lib/Scenes/Artwork/Components/ArtworkActions.tsx]: trivial: typo here on `arist => arist.name`

**Comment by kierangillen** [comment]
@ashfurrow Feel free to merge, lgtm!

**Comment by kierangillen** [comment]
And nice job with this fix we would have otherwise overlooked!

**Comment by artsyit** [comment]
:rocket: PR was released in v1.18.21 :rocket:

**Pull request merged**